sale_stock_delivery_address/__manifest__.py

```diff
@@ -1,14 +1,14 @@
 # Copyright 2020-21 ForgeFlow S.L.
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
@LoisRForgeFlow Was this really AGPL or LGPL (not consistent between manifest and files) ?

Reply:
It was originally improperly licenced as LGPL but as [pointed and fixed by Pedro](https://github.com/OCA/sale-workflow/commit/540ba84ce0bff8b0672303765dd3e7ad5628c717) it needs to be AGPL to depend on sale_procurement_group_by_line.

Short answer: AGPL